UNRELEASED.yaml

```diff
@@ -1,6 +1,12 @@
 # Unreleased
 
 > Place your changes below this line.
+FIXED:
```

Comment:
Can you change this to **Fixed:**

Reply:
I'll leave it as is for now as it's what the release script expects!